Remove Pet#strip_pet_breed

zehash opened the pull request:
https://www.pivotaltracker.com/n/projects/1108894/stories/101479646
# Fixes issue <101479646>

zehash committed:
Remove Pet#strip_pet_breed

https://www.pivotaltracker.com/n/projects/1108894/stories/101479646

#Fixes issue <101479646>

zero51 commented on app/models/pet.rb:
```diff
@@ -97,8 +97,12 @@ def strip_personalities
+  # and sms's being sent out for cats and other animals. 
+  # Sets breed to empty string instead of null. 
+  def null_pet_to_empty_string
+    self.breed = "" if self.breed.nil?
```
Would prefer `self.breed = '' if breed.nil?`.

zehash committed:
Remove Pet#strip_pet_breed as per instruction

https://www.pivotaltracker.com/n/projects/1108894/stories/101479646

#Fixes issue <101479646>

zero51 commented:
Ready to merge.
